tests/integration/gateway/ready.test.js

```diff
@@ -0,0 +1,42 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2023 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const config = require('../../config');
+const { api } = require('../../helpers/api');
+
+const {
+	badRequestSchema,
+} = require('../../schemas/httpGenerics.schema');
+
+const {
+	readySchema,
+} = require('../../schemas/gateway.schema');
+
+const baseUrl = config.SERVICE_ENDPOINT;
+const endpoint = `${baseUrl}/api/ready`;
+
+describe('Ready API', () => {
+	describe(`GET ${endpoint}`, () => {
+		it('Report readiness -> 200 OK', async () => {
+			const response = await api.get(endpoint);
+			expect(response).toMap(readySchema);
```

Comment:
Please include the fallback scenario for when the microservices are not ready, probably wrapped within a `catch` block!?